src/peep/peep.c

```diff
@@ -1668,6 +1668,9 @@ static void peep_go_to_ride_entrance(rct_peep* peep, rct_ride* ride){
 
 	uint8 shift_multiplier = 21;
 	rct_ride_entry* ride_type = get_ride_entry(ride->subtype);
+	if (ride_type == NULL) {
+		return;
+	}
```

Comment:
It might be better to instead skip the code that uses ride entry and vehicle entry. Let the peep still walk to the exit.